### PR TITLE
docs: refactor getting started section

### PR DESCRIPTION
**Pull request by NicholasDunham**
Goals:

- Simplify Getting Started section
- Move "Interaction with Jina AI Cloud" to a separate page
- Fix grammar

### CODE REVIEW

File: docs/fundamentals/documentarray/serialization.md [hunk 1]
```diff
@@ -116,23 +115,23 @@ da_r.summary()
 ```
 
 ```{tip}
-If you go with default `protcol` and `compress` settings, you can simply use `bytes(da)`, which is more Pythonic.
+If you go with the default `protcol` and `compress` settings, you can simply use `bytes(da)`, which is more Pythonic.
```
Comment (JoanFM) [inline docs/fundamentals/documentarray/serialization.md:118]:
```suggestion
If you go with the default `protocol` and `compress` settings, you can simply use `bytes(da)`, which is more Pythonic.
```

**Review by JoanFM — CHANGES_REQUESTED** [review]

File: docs/fundamentals/documentarray/interaction-cloud.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+(interaction-cloud)=
+# Interaction with Jina AI Cloud
+
+```{important}
```
Comment (JoanFM) [inline docs/fundamentals/documentarray/interaction-cloud.md:4]: @samsja I believe this `note` is not needed right? they come when u install docarray with `pip install docarray`?

File: docs/fundamentals/documentarray/serialization.md [hunk 3]
```diff
@@ -358,43 +357,3 @@ To build a DocumentArray from dataframe,
 df = ...
 da = DocumentArray.from_dataframe(df)
 ```
-
-## From/to cloud
```
Comment (alaeddine-13) [inline docs/fundamentals/documentarray/serialization.md:362]: I think removing this section is breaking for some of our content
In this PR I chose to keep it and document cloud support further in a separate section

File: docs/fundamentals/documentarray/interaction-cloud.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
+(interaction-cloud)=
```
Comment (alaeddine-13) [inline docs/fundamentals/documentarray/interaction-cloud.md:1]: This is duplicate
We already added a cloud support section in this PR
https://github.com/docarray/docarray/pull/697

**Review by JohannesMessner — CHANGES_REQUESTED** [review]

File: docs/fundamentals/documentarray/interaction-cloud.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+The {meth}`~docarray.array.mixins.io.pushpull.PushPullMixin.push` and {meth}`~docarray.array.mixins.io.pushpull.PushPullMixin.pull` methods allow you to serialize a DocumentArray object to Jina AI Cloud and share it across machines.
+
+Imagine you're working on a GPU machine via Google Colab/Jupyter. After preprocessing and embedding, you have everything you need in a DocumentArray. You can easily store it to the cloud via:
+
+```python
+from docarray import DocumentArray
+
+da = DocumentArray(...)  # heavy lifting, processing, GPU tasks...
+da.push('myda123', show_progress=True)
```
[the remark this answers is not in the document]
Comment (JoanFM) [reply docs/fundamentals/documentarray/interaction-cloud.md:16]: yes, let's add it

File: docs/fundamentals/documentarray/interaction-cloud.md (hunk 2 above)
Comment (samsja) [reply docs/fundamentals/documentarray/interaction-cloud.md:4]: yes not needed

**Comment by CatStark** [comment]
The part of the integration with Jina Cloud should have more concrete examples. 
These are users that will not read all the documentation, they just need to quickly know what to do with their data.

Some example of creating a docarray from pdf, image, video should be nice.

I know it's explained on the other sections, but we need an easy onboarding for new-comers that don't need to know the technical details

**Pull request closed without merging**